bounties/other/teedy/2/vulnerability.json

```diff
@@ -0,0 +1,50 @@
+{
+  "PackageVulnerabilityID": "2",
+  "DisclosureDate": "2020-12-05",
+  "AffectedVersionRange": "<=1.8",
+  "Summary": "XSS",
+  "Contributor": {
+    "Discloser": "b1nslashsh",
+    "Fixer": ""
+  },
+  "Package": {
+    "Registry": "other",
+    "Name": "teedy",
+    "URL": "https://demo.teedy.io"
+  },
+  "CWEs": [
+    {
+      "ID": "352",
+      "Description": "Cross-Site Scripting(XSS)"
+    }
+  ],
+  "CVSS": {
+    "Version": "3.1",
+    "AV": "",
+    "AC": "",
+    "PR": "",
+    "UI": "",
+    "S": "",
+    "C": "",
+    "I": "",
+    "A": "",
+    "E": "",
+    "RL": "",
+    "RC": "",
+    "Score": "8.8"
```

Comment:
Too high for a `reflected XSS`. If you're able to show better impact like `account takeover` it would indeed be a good impact rate :+1: